docs/core/deploying/trim-self-contained.md

```diff
@@ -9,7 +9,7 @@ ms.date: 04/03/2020
 
 The [framework-dependent deployment model](index.md#publish-framework-dependent) has been the most successful deployment model since the inception of .NET. In this scenario, the application developer bundles only the application and third-party assemblies with the expectation that the .NET runtime and framework libraries will be available in the client machine. This deployment model continues to be the dominant one in .NET Core as well but there are some scenarios where the framework-dependent model is not optimal. The alternative is to publish a [self-contained application](index.md#publish-self-contained), where the .NET Core runtime and framework are bundled together with the application and third-party assemblies.
 
-The trim-self-contained deployment model is a specialized version of the self-contained deployment model that is optimized to reduce deployment size. Minimizing deployment size is a critical requirement for some client-side scenarios like Blazor applications. Depending on the complexity of the application, only a subset of the framework assemblies are required to run the application. These unused parts of the library are unnecessary and can be trimmed from the packaged application. However, there is a risk that the build time analysis of the application can cause failures at runtime, due to not being able to reliably analyze various problematic code patterns (largely centered on reflection use). Because reliability can't be guaranteed, this deployment model is offered as a preview feature. The build time analysis engine provides warnings to the developer of code patterns that are problematic, with the expectation that these code patterns will be fixed. Where possible, we recommend that you move any runtime reflection dependencies in your application to build time by using code that meets the same requirements.
+The trim-self-contained deployment model is a specialized version of the self-contained deployment model that is optimized to reduce deployment size. Minimizing deployment size is a critical requirement for some client-side scenarios like Blazor applications. Depending on the complexity of the application, only a subset of the framework assemblies is required to run the application. These unused parts of the library are unnecessary and can be trimmed from the packaged application. However, there is a risk that the build time analysis of the application can cause failures at runtime, due to not being able to reliably analyze various problematic code patterns (largely centered on reflection use). Because reliability can't be guaranteed, this deployment model is offered as a preview feature. The build time analysis engine provides warnings to the developer of code patterns that are problematic, with the expectation that these code patterns will be fixed. Where possible, we recommend that you move any runtime reflection dependencies in your application to build time by using code that meets the same requirements.
```

Comment:
...a subset of the framework assemblies *is* required...

@tdykstra Is this change correct?